src/test/resources/application.properties

```diff
@@ -4,3 +4,5 @@ spring.datasource.username=sa
 spring.datasource.password=
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
 spring.sql.init.platform=h2
+
+spring.kafka.bootstrap-servers=localhost:9092
```

Comment:
Isn't this host only for running locally? As in wouldn't it require the specific domain depending on the environment it's running in?

If so, maybe we need to split out the application properties into separate property files per env.

Reply:
wrong file, meant to write in the non test one.